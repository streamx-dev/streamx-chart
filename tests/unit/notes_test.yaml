# yaml-language-server: $schema=https://raw.githubusercontent.com/helm-unittest/helm-unittest/main/schema/helm-testsuite.json

suite: test chart notes
templates:
  - templates/NOTES.txt
tests:
  # @Test
  - it: when rest_ingestion service ingress enabled then host printed in notes
    set:
      rest_ingestion:
        ingress:
          host: rest.ingestion.host
    asserts:
      - matchSnapshot: {}
  # @Test
  - it: when delivery service ingress enabled then host printed in notes
    set:
      delivery:
        service1:
          outputs:
            - name: output1
              service:
                port: 80
                targetPort: http
              ingress:
                enabled: true
                host: service1.host
                path: /testpath
    asserts:
      - matchSnapshot: {}