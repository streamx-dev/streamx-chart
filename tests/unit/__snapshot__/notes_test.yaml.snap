when delivery service ingress enabled then host printed in notes:
  1: |
    raw: |-
      Happy StreamX Helming!
      ---------------------
      Delivery services:
      - http://service1.host
when rest_ingestion service ingress enabled then host printed in notes:
  1: |
    raw: |-
      Happy StreamX Helming!
      ---------------------
      Ingestion services:
      - http://rest.ingestion.host
